### ISSUE TITLE
Seriously? I can't block someone from private messaging me?

### ISSUE THREAD

**Issue by yamikuronue** [apxltd/what-bugs#158]
Is it seriously the case that I have to whitelist the entire forum except one person to prevent that person from messaging me privately?

**Comment by BenLubar** [comment]
Is this something we need to discuss in the staff forum?

**Comment by yamikuronue** [comment]
No. The user did not cross into hostile behavior, they're just annoying.

**Comment by masonwheeler** [comment]
Really hope that's not me...
